sim/warlock/demonology/hand_of_guldan.go

```diff
@@ -0,0 +1,74 @@
+package demonology
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/warlock"
+)
+
+func (demonology *DemonologyWarlock) CurseOfGuldanDebuffAura(target *core.Unit) *core.Aura {
+	return target.GetOrRegisterAura(core.Aura{
+		Label:    "CurseOfGuldan-" + demonology.Label,
+		ActionID: core.ActionID{SpellID: 86000},
+		Duration: time.Second * 15,
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			//TODO: Implement Crit rating for pet vs this target only
```

Comment:
We dont support this atm but It can be added to the attack tables as a bonus to crit rating (similiar to how we have extra damage dealt/taken multipliers in the attack tables)